src/models/user.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Entity,
+  model,
+  property
+} from "@loopback/repository";
+
+@model()
+export class User extends Entity {
+  @property({
+    description: "The ID for a user.",
+    id: true,
+    generated: true,
+    type: "number"
+  })
+  id: number;
+  @property({
```

Comment:
Please add an empty line between property definitions to visually delimit where one property ends and another starts.

```ts
// ...
id: number;

@property({
//...
```